src/angular-app/languageforge/lexicon/core/lexicon-project.service.ts

```diff
@@ -100,10 +100,6 @@ export class LexiconProjectService {
     return this.api.call('lex_project_update', [settings], callback);
   }
 
-  updateSettings(smsSettings: any, emailSettings: any, callback?: JsonRpcCallback) {
-    return this.api.call('project_updateSettings', [smsSettings, emailSettings], callback);
-  }
-
```

Comment:
removing this api call looks suspicious to me, mostly because the name of the api doesn't mention anything about sms or email, just the parameters.

There is a big rabbit trail here - if you remove the client side api calling code, should you not also remove the api code in the server?  It goes on and on I think.

Reply:
ok, I've done more research here and the updateSettings and readSettings are dead wood - not used anywhere but the end points still exist.

Please delete both of these methods in the AngularJS client (you've done one here) and delete traces of this back into PHP and PHP tests.  Thanks!

Reply:
done in [c34a6b6](https://github.com/sillsdev/web-languageforge/pull/1611/commits/c34a6b62da664b8d2bc08e7a13096984be0b2ea1)

Reply:
great!